### ISSUE TITLE
Localization errors.

### ISSUE THREAD

**Issue by alexmlt1971** [iina/iina#4729]
**System and IINA version:**

- macOS 14.2.1
- IINA Latest night build 669db3d

**Expected behavior:**
Full application localization

**Actual behavior:**
The localization service for Russian and Ukrainian languages has 100% translation. But there are still places in the application without translation. See screenshots.

<details>
<summary><b>Crash report:</b></summary>

```
```
</details>

<br>

<details>
<summary><b>mpv log:</b></summary>

```
```
</details>

<br>

**Steps to reproduce:**

- [ ] MPV does not have this problem.

**How often does this happen?**
Anytime
<img width="1280" alt="Снимок экрана 2023-12-21 в 15 57 08" src="https://github.com/iina/iina/assets/139448187/ed6c5aaf-b06c-4b63-9909-badacb00eb11">
<img width="1280" alt="Снимок экрана 2023-12-21 в 15 56 52" src="https://github.com/iina/iina/assets/139448187/1db02711-9ce9-4111-8e4c-d6c2ba46ea7b">
<img width="1280" alt="Снимок экрана 2023-12-21 в 15 56 16" src="https://github.com/iina/iina/assets/139448187/fb794dcf-f48f-4b7d-b97e-5fe61c5814cd">

**Comment by low-batt** [comment]
Thank you for the excellent clear report!

I believe there are fixes pending for the first two. I don't think the last one has been reported before. I will look into that one. Caught me about to contact other developers regarding merging localization fixes. Fixing localization issues is a priority.

**Comment by uiryuu** [comment]
Thanks for filing this issue. We cannot guarantee that the release version always contains all the upstream (Crowdin) changes. We have to sync the changes before each IINA release. So what you see in the IINA 1.3.3 is the state of Crowdin when we release IINA 1.3.3, and the Russian and Ukrainian were not 100% at that time. So most of the issue you mentioned will be fixed in the next release. Except for the audio device text, we didn't make that translatable yet. So stay tuned!

**Comment by low-batt** [comment]
I reviewed the situation with `ru` and `uk` translations now that `develop` has been updated with fixes and translations. Some of the fixes to translations as well as a new OSC button that was just merged require new translations. Checking [IINA's instance of Crowdin](https://translate.iina.io/) the `ru` and `uk` translations are currently listed as 99%.

In `Localizable.strings` both `ru` and `uk` translations show some of the OSD messages that were added a while back still lack translations:
```text
"osd.cannot_connect" = "Cannot connect";
"osd.timed_out" = "Timed Out";
```

The new OSC button that was just added:
```text
"osc_toolbar.screenshot" = "Take a screenshot"; 
```

In `PrefUIViewController.strings`  both `ru` and `uk` translations show this, which was shown in the 2nd screen shot above:
```text
/* Class = "NSButtonCell"; title = "Always show float on top status in the title bar"; ObjectID = "ih8-5a-PYY"; */
"ih8-5a-PYY.title" = "Always show float on top status in the title bar";
```

In `PrefControlViewController.strings` both `ru` and `uk` translations show this:
```text
/* Class = "NSTextFieldCell"; title = "Force Touch to:"; ObjectID = "dpH-PO-WKt"; */
"dpH-PO-WKt.title" = "Force Touch:";
```

None of the strings in the `PrefPluginPermissionView.xib` are localized because localization has not been enabled. 

@uiryuu Should we enable that now to get a head start on translations for that feature or wait for 1.4.0?

As for `ru` and `uk` translations I think the next step is to bring translations to 100% on [Crowdin](https://translate.iina.io/), merge the new translations and then check again for missing translations.
